content/doc/developer/tutorial/prepare.adoc

```diff
@@ -16,23 +16,22 @@ references:
 == Download and install a JDK
 
 // TIMEBASED
-Jenkins is based on Java, and to build Jenkins plugins, you need to install a Java Development Kit, or JDK.
-Recent Jenkins releases require JDK 8 to run, so that's what we're using here.
+Jenkins is based on Java, so to build Jenkins plugins you need to install a Java Development Kit (JDK).
+Recent Jenkins releases require JDK 8 to run, so that's what we're using in this tutorial.
 
-Download JDK 8 for free from link:https://www.oracle.com/technetwork/java/javase/downloads/[the Oracle website] and install it.
+You can download and install JDK 8 from link:https://www.oracle.com/technetwork/java/javase/downloads/[the Oracle website].
```

Comment:
Since we're modifying this file already, would you be willing to replace the Oracle reference with a link to one of the Java Development Kits that don't have commercial use restrictions.
``suggestion
You can download and install JDK 8 from https://adoptopenjdk.net/[Adoptium], https://docs.aws.amazon.com/corretto/latest/corretto-8-ug/downloads-list.html[Amazon Corretto]. or the https://developers.redhat.com/products/openjdk/download[Red Hat build of OpenJDK].
Other JDK 8 implementations are also known to work with Jenkins.
```

Reply:
"...[Amazon Corretto]. or the ..." --> It seems that there's a **period** here, but there should be a **comma** "...[Amazon Corretto], or the ..."